openspec/changes/profile-04-safe-project-artifact-writes/specs/module-owned-ide-prompts/spec.md

```
@@ -0,0 +1,14 @@
+## ADDED Requirements
+
+### Requirement: Core materialization of module-owned IDE assets SHALL use safe project writes
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)